apps/bfd-model/bfd-model-rda/src/main/java/gov/cms/bfd/model/rda/PreAdjMcsClaim.java

```diff
@@ -0,0 +1,106 @@
+package gov.cms.bfd.model.rda;
+
+import java.time.LocalDate;
+import java.util.HashSet;
+import java.util.Set;
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.Id;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.experimental.FieldNameConstants;
+
+@Entity
+@Getter
+@Setter
+@EqualsAndHashCode(onlyExplicitlyIncluded = true)
+@FieldNameConstants
+@Table(name = "`McsClaims`", schema = "`pre_adj`")
+public class PreAdjMcsClaim {
+  @Id
+  @Column(name = "`idrClmHdIcn`", length = 15, nullable = false)
+  @EqualsAndHashCode.Include
+  private String idrClmHdIcn;
+
+  @Column(name = "`idrContrId`", length = 5, nullable = false)
+  private String idrContrId;
+
+  @Column(name = "`idrHic`", length = 12, nullable = false)
+  private String idrHic;
+
+  @Column(name = "`idrClaimType`", length = 1, nullable = false)
+  private String idrClaimType;
+
+  @Column(name = "`idrDtlCnt`")
+  private Short idrDtlCnt;
+
+  @Column(name = "`idrBeneLast_1_6`", length = 6)
+  private String idrBeneLast_1_6;
+
+  @Column(name = "`idrBeneFirstInit`", length = 1)
+  private String idrBeneFirstInit;
+
+  @Column(name = "`idrBeneMidInit`", length = 1)
+  private String idrBeneMidInit;
+
+  @Column(name = "`idrBeneSex`", length = 1)
+  private String idrBeneSex;
+
+  @Column(name = "`idrStatusCode`", length = 1)
+  private String idrStatusCode;
+
+  @Column(name = "`idrStatusDate`")
+  private LocalDate idrStatusDate;
+
+  @Column(name = "`idrBillProvNp_1_6`", length = 10)
```

Comment:
is this supposed to be `IDR-BILL-PROV-NPI`?

Reply:
Yes.  Fixed.